src/main/java/com/github/rholder/retry/Retryer.java

```diff
@@ -47,101 +44,119 @@
     private final WaitStrategy waitStrategy;
     private final BlockStrategy blockStrategy;
     private final AttemptTimeLimiter attemptTimeLimiter;
-    private final List<Predicate<Attempt<?>>> rejectionPredicates;
+    private final List<Predicate<Attempt<?>>> retryPredictes;
     private final Collection<RetryListener> listeners;
 
     /**
-     * @param attemptTimeLimiter  to prevent from any single attempt from spinning infinitely
-     * @param stopStrategy        the strategy used to decide when the retryer must stop retrying
-     * @param waitStrategy        the strategy used to decide how much time to sleep between attempts
-     * @param blockStrategy       the strategy used to decide how to block between retry attempts;
-     *                            eg, Thread#sleep(), latches, etc.
-     * @param rejectionPredicates the predicates used to decide if the attempt must be rejected
-     *                            or not. If an attempt is rejected, the retryer will retry the call,
-     *                            unless the stop strategy indicates otherwise or the thread is
-     *                            interrupted.
-     * @param listeners           collection of retry listeners
+     * @param attemptTimeLimiter to prevent from any single attempt from spinning infinitely
+     * @param stopStrategy       the strategy used to decide when the retryer must stop retrying
+     * @param waitStrategy       the strategy used to decide how much time to sleep between attempts
+     * @param blockStrategy      the strategy used to decide how to block between retry attempts;
+     *                           eg, Thread#sleep(), latches, etc.
+     * @param retryPredictes     the predicates used to decide if the attempt must be retried (without
+     *                           regard to the StopStrategy).
+     * @param listeners          collection of retry listeners
      */
     Retryer(@Nonnull AttemptTimeLimiter attemptTimeLimiter,
             @Nonnull StopStrategy stopStrategy,
             @Nonnull WaitStrategy waitStrategy,
             @Nonnull BlockStrategy blockStrategy,
-            @Nonnull List<Predicate<Attempt<?>>> rejectionPredicates,
+            @Nonnull List<Predicate<Attempt<?>>> retryPredictes,
             @Nonnull Collection<RetryListener> listeners) {
         Preconditions.checkNotNull(attemptTimeLimiter, "timeLimiter may not be null");
         Preconditions.checkNotNull(stopStrategy, "stopStrategy may not be null");
         Preconditions.checkNotNull(waitStrategy, "waitStrategy may not be null");
         Preconditions.checkNotNull(blockStrategy, "blockStrategy may not be null");
-        Preconditions.checkNotNull(rejectionPredicates, "rejectionPredicates may not be null");
+        Preconditions.checkNotNull(retryPredictes, "rejectionPredicates may not be null");
```

Comment:
Another high-value comment :) rejectionPredicates --> retryPredicates in the message.